crates/pathfinder/src/sequencer/reply.rs

```diff
@@ -384,6 +388,32 @@ pub mod transaction {
         pub version: TransactionVersion,
     }
 
+    /// Represents deserialized L2 deploy account transaction data.
+    #[serde_as]
+    #[derive(Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
+    #[serde(deny_unknown_fields)]
+    pub struct DeployAccountTransaction {
+        pub contract_address: ContractAddress,
+
+        // COMMON_TXN_PROPERTIES
```

Comment:
Please remove these references to type names from the the JSON-RPC spec because they are misleading. This type is the representation of the response from the Starknet gateway which is unrelated to the JSON-RPC specification.